README.md

```diff
@@ -16,11 +16,16 @@ transmitting Clojure data in a more compact form.
 
 ## Installation
 
-Library releases are published on Clojars. To use the latest version with
-Leiningen, add the following dependency to your project definition:
+Library releases are published on Clojars. To use the latest version,
+add the following dependency to your project definition:
 
-[![Clojars Project](http://clojars.org/mvxcvi/clj-cbor/latest-version.svg)](http://clojars.org/mvxcvi/clj-cbor)
```

Comment:
I like addition of the tools.deps instructions, but I'm loathe to lose the automatic release version from the banner here. Maybe clojars has a tools.deps-style banner we could show too? If not, I'd prefer the tools.deps section to use a "RELEASE" placeholder or instruct the reader to use the version shown here.